test/integration/testdata/golden.v2.json

```diff
@@ -1265,20 +1265,52 @@
       "maxLength": 5,
       "minLength": 1,
       "pattern": "^a.*b$"
+     },
+     "celField": {
+      "type": "string",
+      "default": "",
+      "x-kubernetes-validations": [
+       {
+        "message": "string message",
+        "rule": "self.length() \u003e 0"
```

Comment:
Do we want characters encoded like this?

Reply:
nice catch. definitely not

Reply:
According to [the docs](https://pkg.go.dev/encoding/json#Marshal) this is expected behavior:

> String values encode as JSON strings coerced to valid UTF-8, replacing invalid bytes with the Unicode replacement rune. So that the JSON will be safe to embed inside HTML <script> tags, the string is encoded using HTMLEscape, which replaces "<", ">", "&", U+2028, and U+2029 are escaped to "\u003c","\u003e", "\u0026", "\u2028", and "\u2029". This replacement can be disabled when using an Encoder, by calling SetEscapeHTML(false).

Example:

```go
func TestMarshalString(t *testing.T) {
	myString := "2 > 1"
	myJSON, err := json.Marshal(myString)
	if err != nil {
		t.Fatal(err)
	}
	if `"2 \u003e 1"` != string(myJSON) {
		t.Fatalf("expected %q, got %q", `"2 \u003e 1"`, string(myJSON))
	}
}
```


Given that it is valid JSON and parses the same way, I think it's fine.